crates/ruff_server/src/server.rs

```diff
@@ -60,6 +61,42 @@ impl Server {
 
         conn.initialize_finish(id, initialize_data)?;
 
+        // Register capabilities
+        conn.sender
+            .send(lsp_server::Message::Request(lsp_server::Request {
+                id: 1.into(),
```

Comment:
Hardcoding 1 here seems a bit odd. It might be overkill for now, but should we use an `AtomicUsize` to get the next request id (that can be used when sending other requests too)